spec/timezone.html

```diff
@@ -810,5 +810,34 @@ <h1>
         1. Return *false*.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-parsetimezoneidentifier" type="abstract operation">
+      <h1>
+        ParseTimeZoneIdentifier (
+          _identifier_: a String
+        ): a Record containing [[Name]] and [[OffsetMinutes]] fields, or ~unmatched~
```

Comment:
Thanks for your thorough review!  I'll commit the rest of your suggestions, but wanted to dig in on this one a bit more.

The switch to [[OffsetMinutes]] was intended to align with the main goal of this PR: to enable a Temporal.TimeZone to be stored in <=16 bits.  Unlike some optimizations that some implementations will use and others ignore, I assume that all native implementations will store minutes not nanoseconds.

Given that implementations will be doing the conversion in their own implementation, I thought it would be better to specify the conversion math so that the spec would align more closely with how it will be implemented. In particular I was sensitive to feedback from Shu that Temporal implementations were so large and complex that the only scalable way to code-review them is to ensure that they matched the spec as closely as possible.

So I actually think that specifying the slot as minutes will lead to _less_ buggy implementations, not more.

Also, I liked how using [[OffsetMinutes]] prevents readers/reviewers/implementers from having to worry about accidentally introducing out-of-range data via nanoseconds sneaking in. I generally like it when validation is baked into the data type and the name, as opposed to having to remember and reason about implicit constraints that are not recorded in the type or the name, like "this may be a nanoseconds value that represents an integer number of minutes".

What do you think?

Another kinda-compromise option (inspired by V1 of this PR): of the 6 places where the minutes=>nanoseconds calculation is done in the current PR text, all but one are formatting minutes into an offset string. There's also only one place (GetOffsetStringFor) where a nanoseconds offset is formatted; the rest are minutes only. So what if we did this something like this?
* Add a `FormatOffsetTimeZoneIdentifier` AO. This would limit minutes=>nanoseconds calculation to only two places: the formatting AO and a single line in `TimeZone.prototype.getPossibleInstantsFor`. This would also provide symmetry with the reverse operation ParseTimeZoneIdentifier.
* Remove FormatTimeZoneOffsetString and inline nanoseconds formatting into `GetOffsetStringFor`. Something like this: (rough draft; needs editing)
```
1. Let _offsetNanoseconds_ be ? GetOffsetNanosecondsFor(_timeZone_, _instant_).
1. Let _subMinuteNanoseconds_ be _offsetNanoseconds_ modulo (6 × 10<sup>10</sup>).
1. Let _minutes_ be  (_offsetNanoseconds_ - _subMinuteNanoseconds_) / (6 × 10<sup>10</sup>).
1. Let _offsetString_ be FormatOffsetTimeZoneIdentifier(_minutes_).
1. If _subMinuteNanoseconds_ = 0, return _offsetSring_.
1. <text to calculate seconds and optional sub-seconds, then concatenate to _offsetString_.
1. Return _offsetString_
```

Reply:
> of the 6 places where the minutes=>nanoseconds calculation is done in the current PR text, all but one are formatting minutes into an offset string

Actually, I missed three: `LocalTime`, `UTC`, and `TimeZoneString` in existing 262. What was nice was that this error was pretty obvious because the field names were wrong. ;-)

So switching the formatting method to minutes would only clean up 5 of 9 uses of  `× 10<sup>6</sup`, not 5 of 6 like I thought before. Still might be useful DRY though.

Reply:
> So what if we did this something like this?
> 
> * Add a `FormatOffsetTimeZoneIdentifier` AO. This would limit minutes=>nanoseconds calculation to only two places: the formatting AO and a single line in `TimeZone.prototype.getPossibleInstantsFor`. This would also provide symmetry with the reverse operation `ParseTimeZoneIdentifier`.
> * Remove `FormatTimeZoneOffsetString` and inline sub-minute formatting into `GetOffsetStringFor`.

Yes, that seems pretty good!

Reply:
OK, I made the changes above. Take a look at the new text and let me know what you think.